test/test-api.js

```diff
@@ -244,21 +245,11 @@ var launchTests = function () {
   });
 };
 
-// FIXME: is there a good way to make a server get a port for testing?
 var server;
-var portNumber = 8000;
 var startServer = function () {
-  server = camp.start({port:portNumber, documentRoot:'./test/web'});
+  server = camp.start({port: 0, documentRoot: './test/web'});
   server.on('listening', launchTests);
 };
 var serverStartDomain = require('domain').create();
-serverStartDomain.on('error', function (err) {
-  if (err.code === 'EADDRINUSE') {
-    portNumber++;
-    serverStartDomain.run(startServer);
-  } else {
-    throw err;
```

Comment:
Are you certain that `serverStartDomain` errors are automatically thrown when you don't have a `.on('error', listener)`?

Reply:
No it isn't. Can we just remove the domain wrapper here?

Reply:
Wasn't the domain's only purpose to handle EADDRINUSE? It seems like we can remove the domain altogether.